Description: Upstream changes introduced in version 0.6.2-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 aptitude (0.6.2-2) unstable; urgency=low
 .
   * Backport upstream patch to fix crashes in the resolver. (Closes: #578344)
   * Moved to source version 3.0 (quilt).
 .
 The person named in the Author field signed this changelog entry.
Author: Daniel Burrows <dburrows@debian.org>
Bug-Debian: http://bugs.debian.org/578344

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ aptitude-0.6.2/config.log
@@ -0,0 +1,723 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by aptitude configure 0.6.2, which was
+generated by GNU Autoconf 2.65.  Invocation command line was
+
+  $ ./configure --prefix=/usr --mandir=${prefix}/share/man --infodir=${prefix}/share/info --disable-private-glade-file --disable-docs
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = windrider
+uname -m = x86_64
+uname -r = 2.6.32-3-amd64
+uname -s = Linux
+uname -v = #1 SMP Wed Feb 24 18:07:42 UTC 2010
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /home/daniel/bin
+PATH: /usr/lib/ccache
+PATH: /home/daniel/binary-programs/android-sdk-linux/tools//usr/local/bin
+PATH: /usr/bin
+PATH: /bin
+PATH: /usr/bin/X11
+PATH: /usr/games
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = windrider
+uname -m = x86_64
+uname -r = 2.6.32-3-amd64
+uname -s = Linux
+uname -v = #1 SMP Wed Feb 24 18:07:42 UTC 2010
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /home/daniel/bin
+PATH: /usr/lib/ccache
+PATH: /home/daniel/binary-programs/android-sdk-linux/tools//usr/local/bin
+PATH: /usr/bin
+PATH: /bin
+PATH: /usr/bin/X11
+PATH: /usr/games
+configure:2448: checking for a BSD-compatible install
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2516: result: /usr/bin/X11/install -c
+configure:2527: checking whether build environment is sane
+configure:2448: checking for a BSD-compatible install
+configure:2516: result: /usr/bin/install -c
+configure:2527: checking whether build environment is sane
+configure:2564: result: yes
+configure:2626: checking for gawk
+configure:2642: found /usr/bin/gawk
+configure:2653: result: gawk
+configure:2664: checking whether make sets $(MAKE)
+configure:2686: result: yes
+configure:2564: result: yes
+configure:2626: checking for gawk
+configure:2642: found /usr/bin/gawk
+configure:2653: result: gawk
+configure:2664: checking whether make sets $(MAKE)
+configure:2932: checking for g++
+configure:2948: found /usr/lib/ccache/g++
+configure:2959: result: g++
+configure:2986: checking for C++ compiler version
+configure:2995: g++ --version >&5
+g++ (Debian 4.4.3-7) 4.4.3
+Copyright (C) 2009 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:3006: $? = 0
+configure:2995: g++ -v >&5
+configure:2686: result: yes
+Using built-in specs.
+Target: x86_64-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.3-7' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --program-suffix=-4.4 --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-plugin --enable-objc-gc --with-arch-32=i486 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
+Thread model: posix
+gcc version 4.4.3 (Debian 4.4.3-7) 
+configure:3006: $? = 0
+configure:2995: g++ -V >&5
+g++: '-V' option must have argument
+configure:3006: $? = 1
+configure:2995: g++ -qversion >&5
+configure:2932: checking for g++
+configure:2948: found /usr/lib/ccache/g++
+configure:2959: result: g++
+configure:2986: checking for C++ compiler version
+configure:2995: g++ --version >&5
+configure:3006: $? = 1
+g++ (Debian 4.4.3-7) 4.4.3
+Copyright (C) 2009 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:3006: $? = 0
+configure:3026: checking whether the C++ compiler works
+configure:2995: g++ -v >&5
+configure:3048: g++ -g -O2   conftest.cpp  >&5
+configure:3006: $? = 0
+configure:2995: g++ -V >&5
+g++: '-V' option must have argument
+configure:3006: $? = 1
+configure:2995: g++ -qversion >&5
+g++: unrecognized option '-qversion'
+g++: no input files
+configure:3006: $? = 1
+configure:3026: checking whether the C++ compiler works
+configure:3048: g++ -g -O2   conftest.cpp  >&5
+configure:3052: $? = 0
+configure:3101: result: yes
+configure:3104: checking for C++ compiler default output file name
+configure:3106: result: a.out
+configure:3112: checking for suffix of executables
+configure:3119: g++ -o conftest -g -O2   conftest.cpp  >&5
+configure:3052: $? = 0
+configure:3101: result: yes
+configure:3104: checking for C++ compiler default output file name
+configure:3106: result: a.out
+configure:3112: checking for suffix of executables
+configure:3119: g++ -o conftest -g -O2   conftest.cpp  >&5
+configure:3123: $? = 0
+configure:3145: result: 
+configure:3167: checking whether we are cross compiling
+configure:3175: g++ -o conftest -g -O2   conftest.cpp  >&5
+configure:3123: $? = 0
+configure:3145: result: 
+configure:3167: checking whether we are cross compiling
+configure:3175: g++ -o conftest -g -O2   conftest.cpp  >&5
+configure:3179: $? = 0
+configure:3186: ./conftest
+configure:3190: $? = 0
+configure:3205: result: no
+configure:3210: checking for suffix of object files
+configure:3232: g++ -c -g -O2  conftest.cpp >&5
+configure:3179: $? = 0
+configure:3186: ./conftest
+configure:3190: $? = 0
+configure:3205: result: no
+configure:3210: checking for suffix of object files
+configure:3232: g++ -c -g -O2  conftest.cpp >&5
+configure:3236: $? = 0
+configure:3257: result: o
+configure:3261: checking whether we are using the GNU C++ compiler
+configure:3280: g++ -c -g -O2  conftest.cpp >&5
+configure:3236: $? = 0
+configure:3257: result: o
+configure:3261: checking whether we are using the GNU C++ compiler
+configure:3280: g++ -c -g -O2  conftest.cpp >&5
+configure:3280: $? = 0
+configure:3289: result: yes
+configure:3298: checking whether g++ accepts -g
+configure:3280: $? = 0
+configure:3289: result: yes
+configure:3298: checking whether g++ accepts -g
+configure:3318: g++ -c -g  conftest.cpp >&5
+configure:3318: g++ -c -g  conftest.cpp >&5
+configure:3318: $? = 0
+configure:3359: result: yes
+configure:3393: checking for style of include used by make
+configure:3318: $? = 0
+configure:3359: result: yes
+configure:3393: checking for style of include used by make
+configure:3421: result: GNU
+configure:3449: checking dependency style of g++
+configure:3421: result: GNU
+configure:3449: checking dependency style of g++
+configure:3539: result: gcc3
+configure:3539: result: gcc3
+configure:3599: checking for ranlib
+configure:3615: found /usr/bin/ranlib
+configure:3626: result: ranlib
+configure:3650: checking for po4a
+configure:3599: checking for ranlib
+configure:3668: found /usr/bin/po4a
+configure:3680: result: /usr/bin/po4a
+configure:3615: found /usr/bin/ranlib
+configure:3626: result: ranlib
+configure:3716: checking for initscr in -lncursesw
+configure:3650: checking for po4a
+configure:3668: found /usr/bin/po4a
+configure:3680: result: /usr/bin/po4a
+configure:3716: checking for initscr in -lncursesw
+configure:3741: g++ -o conftest -g -O2   conftest.cpp -lncursesw   >&5
+configure:3741: g++ -o conftest -g -O2   conftest.cpp -lncursesw   >&5
+configure:3741: $? = 0
+configure:3741: $? = 0
+configure:3750: result: yes
+configure:3750: result: yes
+configure:3763: checking for main in -lapt-pkg
+configure:3763: checking for main in -lapt-pkg
+configure:3782: g++ -o conftest -g -O2   conftest.cpp -lapt-pkg  -lncursesw  >&5
+configure:3782: g++ -o conftest -g -O2   conftest.cpp -lapt-pkg  -lncursesw  >&5
+configure:3782: $? = 0
+configure:3791: result: yes
+configure:3782: $? = 0
+configure: failed program was:
+configure:3805: checking whether apt includes the automatic dependency removal patch (required)
+configure:3791: result: no
+configure:3819: g++ -c -g -O2  conftest.cpp >&5
+configure:3801: error: Can't find the APT libraries -- please install libapt-pkg-dev
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+g++: conftest.cpp: No such file or directory
+g++: no input files
+configure:3819: $? = 1
+configure: failed program was:
+configure:3823: error: in `/home/daniel/debian-pkgs/aptitude/aptitude-0.6.2':
+configure:3826: error: need libapt-pkg 0.7 or later
+See `config.log' for more details.
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_c_const=yes
+ac_cv_c_inline=yes
+ac_cv_cxx_compiler_gnu=yes
+ac_cv_env_CCC_set=
+ac_cv_env_CCC_value=
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-g -O2'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CWIDGET_CFLAGS_set=
+ac_cv_env_CWIDGET_CFLAGS_value=
+ac_cv_env_CWIDGET_LIBS_set=
+ac_cv_env_CWIDGET_LIBS_value=
+ac_cv_env_CXXCPP_set=
+ac_cv_env_CXXCPP_value=
+ac_cv_env_CXXFLAGS_set=set
+ac_cv_env_CXXFLAGS_value='-g -O2'
+ac_cv_env_CXX_set=
+ac_cv_env_CXX_value=
+ac_cv_env_GLIBMM_CFLAGS_set=
+ac_cv_env_GLIBMM_CFLAGS_value=
+ac_cv_env_GLIBMM_LIBS_set=
+ac_cv_env_GLIBMM_LIBS_value=
+ac_cv_env_GTHREAD_CFLAGS_set=
+ac_cv_env_GTHREAD_CFLAGS_value=
+ac_cv_env_GTHREAD_LIBS_set=
+ac_cv_env_GTHREAD_LIBS_value=
+ac_cv_env_GTKMM_CFLAGS_set=
+ac_cv_env_GTKMM_CFLAGS_value=
+ac_cv_env_GTKMM_LIBS_set=
+ac_cv_env_GTKMM_LIBS_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBGLADEMM_CFLAGS_set=
+ac_cv_env_LIBGLADEMM_CFLAGS_value=
+ac_cv_env_LIBGLADEMM_LIBS_set=
+ac_cv_env_LIBGLADEMM_LIBS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_LOG4CXX_CFLAGS_set=
+ac_cv_env_LOG4CXX_CFLAGS_value=
+ac_cv_env_LOG4CXX_LIBS_set=
+ac_cv_env_LOG4CXX_LIBS_value=
+ac_cv_env_PKG_CONFIG_set=
+ac_cv_env_PKG_CONFIG_value=
+ac_cv_env_SIGC_CFLAGS_set=
+ac_cv_env_SIGC_CFLAGS_value=
+ac_cv_env_SIGC_LIBS_set=
+ac_cv_env_SIGC_LIBS_value=
+ac_cv_env_SQLITE3_CFLAGS_set=
+ac_cv_env_SQLITE3_CFLAGS_value=
+ac_cv_env_SQLITE3_LIBS_set=
+ac_cv_env_SQLITE3_LIBS_value=
+ac_cv_env_VTE_CFLAGS_set=
+ac_cv_env_VTE_CFLAGS_value=
+ac_cv_env_VTE_LIBS_set=
+ac_cv_env_VTE_LIBS_value=
+ac_cv_env_build_alias_set=
+ac_cv_env_build_alias_value=
+ac_cv_env_ept_CFLAGS_set=
+ac_cv_env_ept_CFLAGS_value=
+ac_cv_env_ept_LIBS_set=
+ac_cv_env_ept_LIBS_value=
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_lib_apt_pkg_main=no
+ac_cv_lib_ncursesw_initscr=yes
+ac_cv_objext=o
+ac_cv_path_PO4A=/usr/bin/po4a
+ac_cv_path_install='/usr/bin/X11/install -c'
+ac_cv_prog_AWK=gawk
+ac_cv_prog_ac_ct_CXX=g++
+ac_cv_prog_ac_ct_RANLIB=ranlib
+ac_cv_prog_cxx_g=yes
+ac_cv_prog_make_make_set=yes
+am_cv_CXX_dependencies_compiler_type=gcc3
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ACLOCAL='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run aclocal-1.9'
+AMDEPBACKSLASH='\'
+AMDEP_FALSE='#'
+AMDEP_TRUE=''
+AMTAR='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run tar'
+AUTOCONF='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run autoconf'
+AUTOHEADER='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run autoheader'
+AUTOMAKE='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run automake-1.9'
+AWK='gawk'
+BOOST_CPPFLAGS=''
+BOOST_IOSTREAMS_LIB=''
+BOOST_LDFLAGS=''
+BUILD_GTK_FALSE=''
+BUILD_GTK_TRUE=''
+CC=''
+CCDEPMODE=''
+CFLAGS='-g -O2'
+CPPFLAGS=''
+CWIDGET_CFLAGS=''
+CWIDGET_LIBS=''
+CXX='g++'
+CXXCPP=''
+CXXDEPMODE='depmode=gcc3'
+CXXFLAGS='-g -O2'
+CYGPATH_W='echo'
+DEFS=''
+DEPDIR='.deps'
+DOCDIRS=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP=''
+EXEEXT=''
+GLIBMM_CFLAGS=''
+GLIBMM_LIBS=''
+GMSGFMT=''
+GREP=''
+GTHREAD_CFLAGS=''
+GTHREAD_LIBS=''
+GTKMM_CFLAGS=''
+GTKMM_LIBS=''
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+INSTALL_STRIP_PROGRAM='${SHELL} $(install_sh) -c -s'
+INTLLIBS=''
+INTL_MACOSX_LIBS=''
+LDFLAGS=''
+LIBGLADEMM_CFLAGS=''
+LIBGLADEMM_LIBS=''
+LIBICONV=''
+LIBINTL=''
+LIBOBJS=''
+LIBS='-lncursesw '
+LOG4CXX_CFLAGS=''
+LOG4CXX_LIBS=''
+LTLIBICONV=''
+LTLIBINTL=''
+LTLIBOBJS=''
+MAKEINFO='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run makeinfo'
+MKINSTALLDIRS=''
+MSGFMT=''
+MSGMERGE=''
+OBJEXT='o'
+PACKAGE='aptitude'
+PACKAGE_BUGREPORT=''
+PACKAGE_NAME='aptitude'
+PACKAGE_STRING='aptitude 0.6.2'
+PACKAGE_TARNAME='aptitude'
+PACKAGE_URL=''
+PACKAGE_VERSION='0.6.2'
+PATH_SEPARATOR=':'
+PKG_CONFIG=''
+PO4A='/usr/bin/po4a'
+POSUB=''
+RANLIB='ranlib'
+SET_MAKE=''
+SHELL='/bin/bash'
+SIGC_CFLAGS=''
+SIGC_LIBS=''
+SQLITE3_CFLAGS=''
+SQLITE3_LIBS=''
+SRCDIRS=''
+STATEDIR=''
+STRIP=''
+USE_NLS=''
+USE_PO4A_FALSE='#'
+USE_PO4A_TRUE=''
+VERSION='0.6.2'
+VTE_CFLAGS=''
+VTE_LIBS=''
+XGETTEXT=''
+ac_ct_CC=''
+ac_ct_CXX='g++'
+am__fastdepCC_FALSE=''
+am__fastdepCC_TRUE=''
+am__fastdepCXX_FALSE='#'
+am__fastdepCXX_TRUE=''
+am__include='include'
+am__leading_dot='.'
+am__quote=''
+am__tar='${AMTAR} chof - "$$tardir"'
+am__untar='${AMTAR} xf -'
+bindir='${exec_prefix}/bin'
+build=''
+build_alias=''
+build_cpu=''
+build_os=''
+build_vendor=''
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
+dvidir='${docdir}'
+ept_CFLAGS=''
+ept_LIBS=''
+exec_prefix='NONE'
+host=''
+host_alias=''
+host_cpu=''
+host_os=''
+host_vendor=''
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+install_sh='/home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/install-sh'
+libdir='${exec_prefix}/lib'
+libexecdir='${exec_prefix}/libexec'
+localedir='${datarootdir}/locale'
+localstatedir='${prefix}/var'
+mandir='${prefix}/share/man'
+mkdir_p='mkdir -p --'
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+sysconfdir='${prefix}/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+ac_cv_c_const=yes
+ac_cv_c_inline=yes
+ac_cv_cxx_compiler_gnu=yes
+ac_cv_env_CCC_set=
+ac_cv_env_CCC_value=
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-g -O2'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CWIDGET_CFLAGS_set=
+ac_cv_env_CWIDGET_CFLAGS_value=
+ac_cv_env_CWIDGET_LIBS_set=
+ac_cv_env_CWIDGET_LIBS_value=
+ac_cv_env_CXXCPP_set=
+ac_cv_env_CXXCPP_value=
+ac_cv_env_CXXFLAGS_set=set
+ac_cv_env_CXXFLAGS_value='-g -O2'
+ac_cv_env_CXX_set=
+ac_cv_env_CXX_value=
+ac_cv_env_GLIBMM_CFLAGS_set=
+ac_cv_env_GLIBMM_CFLAGS_value=
+ac_cv_env_GLIBMM_LIBS_set=
+ac_cv_env_GLIBMM_LIBS_value=
+ac_cv_env_GTHREAD_CFLAGS_set=
+ac_cv_env_GTHREAD_CFLAGS_value=
+ac_cv_env_GTHREAD_LIBS_set=
+ac_cv_env_GTHREAD_LIBS_value=
+ac_cv_env_GTKMM_CFLAGS_set=
+ac_cv_env_GTKMM_CFLAGS_value=
+ac_cv_env_GTKMM_LIBS_set=
+ac_cv_env_GTKMM_LIBS_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBGLADEMM_CFLAGS_set=
+ac_cv_env_LIBGLADEMM_CFLAGS_value=
+ac_cv_env_LIBGLADEMM_LIBS_set=
+ac_cv_env_LIBGLADEMM_LIBS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_LOG4CXX_CFLAGS_set=
+ac_cv_env_LOG4CXX_CFLAGS_value=
+ac_cv_env_LOG4CXX_LIBS_set=
+ac_cv_env_LOG4CXX_LIBS_value=
+ac_cv_env_PKG_CONFIG_set=
+ac_cv_env_PKG_CONFIG_value=
+ac_cv_env_SIGC_CFLAGS_set=
+ac_cv_env_SIGC_CFLAGS_value=
+ac_cv_env_SIGC_LIBS_set=
+ac_cv_env_SIGC_LIBS_value=
+ac_cv_env_SQLITE3_CFLAGS_set=
+ac_cv_env_SQLITE3_CFLAGS_value=
+ac_cv_env_SQLITE3_LIBS_set=
+ac_cv_env_SQLITE3_LIBS_value=
+ac_cv_env_VTE_CFLAGS_set=
+ac_cv_env_VTE_CFLAGS_value=
+ac_cv_env_VTE_LIBS_set=
+ac_cv_env_VTE_LIBS_value=
+ac_cv_env_build_alias_set=
+ac_cv_env_build_alias_value=
+ac_cv_env_ept_CFLAGS_set=
+ac_cv_env_ept_CFLAGS_value=
+ac_cv_env_ept_LIBS_set=
+ac_cv_env_ept_LIBS_value=
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_lib_apt_pkg_main=yes
+ac_cv_lib_ncursesw_initscr=yes
+ac_cv_objext=o
+ac_cv_path_PO4A=/usr/bin/po4a
+ac_cv_path_install='/usr/bin/install -c'
+ac_cv_prog_AWK=gawk
+ac_cv_prog_ac_ct_CXX=g++
+ac_cv_prog_ac_ct_RANLIB=ranlib
+ac_cv_prog_cxx_g=yes
+ac_cv_prog_make_make_set=yes
+am_cv_CXX_dependencies_compiler_type=gcc3
+
+/* confdefs.h */
+#define PACKAGE_NAME "aptitude"
+#define PACKAGE_TARNAME "aptitude"
+#define PACKAGE_VERSION "0.6.2"
+#define PACKAGE_STRING "aptitude 0.6.2"
+#define PACKAGE_BUGREPORT ""
+#define PACKAGE_URL ""
+#define PACKAGE "aptitude"
+#define VERSION "0.6.2"
+#define PACKAGE "aptitude"
+#define VERSION "0.6.2"
+#define HAVE_LIBNCURSESW 1
+#define HAVE_LIBNCURSESW 1
+#define HAVE_LIBAPT_PKG 1
+
+configure: exit 1
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ACLOCAL='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run aclocal-1.9'
+AMDEPBACKSLASH='\'
+AMDEP_FALSE='#'
+AMDEP_TRUE=''
+AMTAR='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run tar'
+AUTOCONF='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run autoconf'
+AUTOHEADER='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run autoheader'
+AUTOMAKE='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run automake-1.9'
+AWK='gawk'
+BOOST_CPPFLAGS=''
+BOOST_IOSTREAMS_LIB=''
+BOOST_LDFLAGS=''
+BUILD_GTK_FALSE=''
+BUILD_GTK_TRUE=''
+CC=''
+CCDEPMODE=''
+CFLAGS='-g -O2'
+CPPFLAGS=''
+CWIDGET_CFLAGS=''
+CWIDGET_LIBS=''
+CXX='g++'
+CXXCPP=''
+CXXDEPMODE='depmode=gcc3'
+CXXFLAGS='-g -O2'
+CYGPATH_W='echo'
+DEFS=''
+DEPDIR='.deps'
+DOCDIRS=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP=''
+EXEEXT=''
+GLIBMM_CFLAGS=''
+GLIBMM_LIBS=''
+GMSGFMT=''
+GREP=''
+GTHREAD_CFLAGS=''
+GTHREAD_LIBS=''
+GTKMM_CFLAGS=''
+GTKMM_LIBS=''
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+INSTALL_STRIP_PROGRAM='${SHELL} $(install_sh) -c -s'
+INTLLIBS=''
+INTL_MACOSX_LIBS=''
+LDFLAGS=''
+LIBGLADEMM_CFLAGS=''
+LIBGLADEMM_LIBS=''
+LIBICONV=''
+LIBINTL=''
+LIBOBJS=''
+LIBS='-lapt-pkg -lncursesw '
+LOG4CXX_CFLAGS=''
+LOG4CXX_LIBS=''
+LTLIBICONV=''
+LTLIBINTL=''
+LTLIBOBJS=''
+MAKEINFO='${SHELL} /home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/missing --run makeinfo'
+MKINSTALLDIRS=''
+MSGFMT=''
+MSGMERGE=''
+OBJEXT='o'
+PACKAGE='aptitude'
+PACKAGE_BUGREPORT=''
+PACKAGE_NAME='aptitude'
+PACKAGE_STRING='aptitude 0.6.2'
+PACKAGE_TARNAME='aptitude'
+PACKAGE_URL=''
+PACKAGE_VERSION='0.6.2'
+PATH_SEPARATOR=':'
+PKG_CONFIG=''
+PO4A='/usr/bin/po4a'
+POSUB=''
+RANLIB='ranlib'
+SET_MAKE=''
+SHELL='/bin/bash'
+SIGC_CFLAGS=''
+SIGC_LIBS=''
+SQLITE3_CFLAGS=''
+SQLITE3_LIBS=''
+SRCDIRS=''
+STATEDIR=''
+STRIP=''
+USE_NLS=''
+USE_PO4A_FALSE='#'
+USE_PO4A_TRUE=''
+VERSION='0.6.2'
+VTE_CFLAGS=''
+VTE_LIBS=''
+XGETTEXT=''
+ac_ct_CC=''
+ac_ct_CXX='g++'
+am__fastdepCC_FALSE=''
+am__fastdepCC_TRUE=''
+am__fastdepCXX_FALSE='#'
+am__fastdepCXX_TRUE=''
+am__include='include'
+am__leading_dot='.'
+am__quote=''
+am__tar='${AMTAR} chof - "$$tardir"'
+am__untar='${AMTAR} xf -'
+bindir='${exec_prefix}/bin'
+build=''
+build_alias=''
+build_cpu=''
+build_os=''
+build_vendor=''
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
+dvidir='${docdir}'
+ept_CFLAGS=''
+ept_LIBS=''
+exec_prefix='NONE'
+host=''
+host_alias=''
+host_cpu=''
+host_os=''
+host_vendor=''
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+install_sh='/home/daniel/debian-pkgs/aptitude/aptitude-0.6.2/install-sh'
+libdir='${exec_prefix}/lib'
+libexecdir='${exec_prefix}/libexec'
+localedir='${datarootdir}/locale'
+localstatedir='${prefix}/var'
+mandir='${prefix}/share/man'
+mkdir_p='mkdir -p --'
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+sysconfdir='${prefix}/etc'
+target_alias=''
+
+configure: exit 1
--- /dev/null
+++ aptitude-0.6.2/po/POTFILES.in.in
@@ -0,0 +1,307 @@
+src/apt_config_treeitems.cc
+src/apt_config_treeitems.h
+src/apt_config_widgets.cc
+src/apt_config_widgets.h
+src/apt_info_tree.cc
+src/apt_info_tree.h
+src/aptitude.h
+src/apt_options.cc
+src/apt_options.h
+src/apt_undo_tree.cc
+src/apt_undo_tree.h
+src/broken_indicator.cc
+src/broken_indicator.h
+src/cmdline/cmdline_action.cc
+src/cmdline/cmdline_action.h
+src/cmdline/cmdline_changelog.cc
+src/cmdline/cmdline_changelog.h
+src/cmdline/cmdline_check_resolver.cc
+src/cmdline/cmdline_check_resolver.h
+src/cmdline/cmdline_clean.cc
+src/cmdline/cmdline_clean.h
+src/cmdline/cmdline_common.h
+src/cmdline/cmdline_do_action.cc
+src/cmdline/cmdline_do_action.h
+src/cmdline/cmdline_download.cc
+src/cmdline/cmdline_download.h
+src/cmdline/cmdline_dump_resolver.cc
+src/cmdline/cmdline_dump_resolver.h
+src/cmdline/cmdline_extract_cache_subset.cc
+src/cmdline/cmdline_extract_cache_subset.h
+src/cmdline/cmdline_forget_new.cc
+src/cmdline/cmdline_forget_new.h
+src/cmdline/cmdline_main_loop.cc
+src/cmdline/cmdline_main_loop.h
+src/cmdline/cmdline_moo.cc
+src/cmdline/cmdline_moo.h
+src/cmdline/cmdline_progress.cc
+src/cmdline/cmdline_progress.h
+src/cmdline/cmdline_prompt.cc
+src/cmdline/cmdline_prompt.h
+src/cmdline/cmdline_resolver.cc
+src/cmdline/cmdline_resolver.h
+src/cmdline/cmdline_search.cc
+src/cmdline/cmdline_search.h
+src/cmdline/cmdline_show_broken.cc
+src/cmdline/cmdline_show_broken.h
+src/cmdline/cmdline_show.cc
+src/cmdline/cmdline_show.h
+src/cmdline/cmdline_simulate.cc
+src/cmdline/cmdline_simulate.h
+src/cmdline/cmdline_spinner.cc
+src/cmdline/cmdline_spinner.h
+src/cmdline/cmdline_update.cc
+src/cmdline/cmdline_update.h
+src/cmdline/cmdline_user_tag.cc
+src/cmdline/cmdline_user_tag.h
+src/cmdline/cmdline_util.cc
+src/cmdline/cmdline_util.h
+src/cmdline/cmdline_versions.cc
+src/cmdline/cmdline_versions.h
+src/cmdline/cmdline_why.cc
+src/cmdline/cmdline_why.h
+src/defaults.cc
+src/defaults.h
+src/dep_item.cc
+src/dep_item.h
+src/desc_render.cc
+src/desc_render.h
+src/download_item.cc
+src/download_item.h
+src/download_list.cc
+src/download_list.h
+src/download_thread.cc
+src/download_thread.h
+src/edit_pkg_hier.cc
+src/edit_pkg_hier.h
+src/generic/apt/acqprogress.cc
+src/generic/apt/acqprogress.h
+src/generic/apt/aptcache.cc
+src/generic/apt/aptcache.h
+src/generic/apt/apt.cc
+src/generic/apt/apt.h
+src/generic/apt/aptitudepolicy.cc
+src/generic/apt/aptitudepolicy.h
+src/generic/apt/aptitude_resolver.cc
+src/generic/apt/aptitude_resolver_cost_settings.cc
+src/generic/apt/aptitude_resolver_cost_settings.h
+src/generic/apt/aptitude_resolver_cost_syntax.cc
+src/generic/apt/aptitude_resolver_cost_syntax.h
+src/generic/apt/aptitude_resolver_cost_types.cc
+src/generic/apt/aptitude_resolver_cost_types.h
+src/generic/apt/aptitude_resolver.h
+src/generic/apt/aptitude_resolver_universe.cc
+src/generic/apt/aptitude_resolver_universe.h
+src/generic/apt/apt_undo_group.cc
+src/generic/apt/apt_undo_group.h
+src/generic/apt/changelog_parse.cc
+src/generic/apt/changelog_parse.h
+src/generic/apt/config_signal.cc
+src/generic/apt/config_signal.h
+src/generic/apt/desc_parse.cc
+src/generic/apt/desc_parse.h
+src/generic/apt/download_install_manager.cc
+src/generic/apt/download_install_manager.h
+src/generic/apt/download_manager.cc
+src/generic/apt/download_manager.h
+src/generic/apt/download_queue.cc
+src/generic/apt/download_queue.h
+src/generic/apt/download_signal_log.cc
+src/generic/apt/download_signal_log.h
+src/generic/apt/download_update_manager.cc
+src/generic/apt/download_update_manager.h
+src/generic/apt/dump_packages.cc
+src/generic/apt/dump_packages.h
+src/generic/apt/history/history_entry.cc
+src/generic/apt/history/history_entry.h
+src/generic/apt/infer_reason.cc
+src/generic/apt/infer_reason.h
+src/generic/apt/log.cc
+src/generic/apt/log.h
+src/generic/apt/matching/compare_patterns.cc
+src/generic/apt/matching/compare_patterns.h
+src/generic/apt/matching/match.cc
+src/generic/apt/matching/match.h
+src/generic/apt/matching/parse.cc
+src/generic/apt/matching/parse.h
+src/generic/apt/matching/pattern.cc
+src/generic/apt/matching/pattern.h
+src/generic/apt/matching/serialize.cc
+src/generic/apt/matching/serialize.h
+src/generic/apt/parse_dpkg_status.cc
+src/generic/apt/parse_dpkg_status.h
+src/generic/apt/pkg_acqfile.cc
+src/generic/apt/pkg_acqfile.h
+src/generic/apt/pkg_changelog.cc
+src/generic/apt/pkg_changelog.h
+src/generic/apt/pkg_hier.cc
+src/generic/apt/pkg_hier_dump.cc
+src/generic/apt/pkg_hier.h
+src/generic/apt/resolver_manager.cc
+src/generic/apt/resolver_manager.h
+src/generic/apt/rev_dep_iterator.h
+src/generic/apt/screenshot.cc
+src/generic/apt/screenshot.h
+src/generic/apt/tags.cc
+src/generic/apt/tags.h
+src/generic/apt/tasks.cc
+src/generic/apt/tasks.h
+src/generic/problemresolver/choice.h
+src/generic/problemresolver/choice_indexed_map.h
+src/generic/problemresolver/choice_set.h
+src/generic/problemresolver/cost.cc
+src/generic/problemresolver/cost.h
+src/generic/problemresolver/cost_limits.cc
+src/generic/problemresolver/cost_limits.h
+src/generic/problemresolver/dummy_universe.cc
+src/generic/problemresolver/dummy_universe.h
+src/generic/problemresolver/dump_universe.h
+src/generic/problemresolver/exceptions.h
+src/generic/problemresolver/incremental_expression.cc
+src/generic/problemresolver/incremental_expression.h
+src/generic/problemresolver/problemresolver.h
+src/generic/problemresolver/promotion_set.h
+src/generic/problemresolver/resolver_undo.h
+src/generic/problemresolver/sanity_check_universe.h
+src/generic/problemresolver/search_graph.h
+src/generic/problemresolver/solution.h
+src/generic/problemresolver/test.cc
+src/generic/util/compare3.h
+src/generic/util/dense_setset.h
+src/generic/util/dirent_safe.h
+src/generic/util/file_cache.cc
+src/generic/util/file_cache.h
+src/generic/util/immlist.h
+src/generic/util/immset.h
+src/generic/util/job_queue_thread.h
+src/generic/util/maybe.h
+src/generic/util/mut_fun.h
+src/generic/util/parsers.h
+src/generic/util/post_thunk.h
+src/generic/util/refcounted_base.cc
+src/generic/util/refcounted_base.h
+src/generic/util/refcounted_wrapper.h
+src/generic/util/safe_slot.h
+src/generic/util/setset.h
+src/generic/util/sqlite.cc
+src/generic/util/sqlite.h
+src/generic/util/temp.cc
+src/generic/util/temp.h
+src/generic/util/undo.cc
+src/generic/util/undo.h
+src/generic/util/util.cc
+src/generic/util/util.h
+src/gtk/aptitude.glade
+src/gtk/area.cc
+src/gtk/area.h
+src/gtk/changelog.cc
+src/gtk/changelog.h
+src/gtk/constants.h
+src/gtk/dashboardtab.cc
+src/gtk/dashboardtab.h
+src/gtk/dependency_chains_tab.cc
+src/gtk/dependency_chains_tab.h
+src/gtk/download.cc
+src/gtk/download.h
+src/gtk/dpkg_terminal.cc
+src/gtk/dpkg_terminal.h
+src/gtk/entitysummary.cc
+src/gtk/entitysummary.h
+src/gtk/entityview.cc
+src/gtk/entityview.h
+src/gtk/errortab.cc
+src/gtk/errortab.h
+src/gtk/filesview.cc
+src/gtk/filesview.h
+src/gtk/gui.cc
+src/gtk/gui.h
+src/gtk/hyperlink.cc
+src/gtk/hyperlink.h
+src/gtk/info.cc
+src/gtk/info.h
+src/gtk/notify.cc
+src/gtk/notify.h
+src/gtk/packageinformation.cc
+src/gtk/packageinformation.h
+src/gtk/packagestab.cc
+src/gtk/packagestab.h
+src/gtk/pkgview.cc
+src/gtk/pkgview.h
+src/gtk/previewtab.cc
+src/gtk/previewtab.h
+src/gtk/progress.cc
+src/gtk/progress.h
+src/gtk/resolver.cc
+src/gtk/resolver.h
+src/gtk/screenshot_cache.cc
+src/gtk/screenshot_cache.h
+src/gtk/screenshot.cc
+src/gtk/screenshot.h
+src/gtk/tab.cc
+src/gtk/tab.h
+src/gtk/treeview_cell_tooltips.cc
+src/gtk/treeview_cell_tooltips.h
+src/gtk/util/property.h
+src/load_config.cc
+src/load_config.h
+src/load_grouppolicy.cc
+src/load_grouppolicy.h
+src/load_pkgview.cc
+src/load_pkgview.h
+src/load_sortpolicy.cc
+src/load_sortpolicy.h
+src/loggers.cc
+src/loggers.h
+src/main.cc
+src/menu_redirect.cc
+src/menu_redirect.h
+src/menu_text_layout.cc
+src/menu_text_layout.h
+src/menu_tree.cc
+src/menu_tree.h
+src/mine/board.cc
+src/mine/board.h
+src/mine/cmine.cc
+src/mine/cmine.h
+src/pkg_columnizer.cc
+src/pkg_columnizer.h
+src/pkg_grouppolicy.cc
+src/pkg_grouppolicy.h
+src/pkg_info_screen.cc
+src/pkg_info_screen.h
+src/pkg_item.cc
+src/pkg_item.h
+src/pkg_item_with_subtree.h
+src/pkg_node.cc
+src/pkg_node.h
+src/pkg_sortpolicy.cc
+src/pkg_sortpolicy.h
+src/pkg_subtree.cc
+src/pkg_subtree.h
+src/pkg_tree.cc
+src/pkg_tree.h
+src/pkg_ver_item.cc
+src/pkg_ver_item.h
+src/pkg_view.cc
+src/pkg_view.h
+src/progress.cc
+src/progress.h
+src/reason_fragment.cc
+src/reason_fragment.h
+src/safe_slot_event.h
+src/solution_dialog.cc
+src/solution_dialog.h
+src/solution_fragment.cc
+src/solution_fragment.h
+src/solution_item.cc
+src/solution_item.h
+src/solution_screen.cc
+src/solution_screen.h
+src/trust.cc
+src/trust.h
+src/ui.cc
+src/ui_download_manager.cc
+src/ui_download_manager.h
+src/ui.h
+src/view_changelog.cc
+src/view_changelog.h
